pc_ble_driver_py/ble_driver.py

```diff
@@ -369,6 +369,63 @@ def to_c(self):
 
         return scan_params
 
+class BLEGapConnSecMode(object):
+    def __init__(self, sm=0, lv=0):
+        self.sm = sm
+        self.lv = lv
+        
+    def set_no_access(self):
+        """BLE_GAP_CONN_SEC_MODE_SET_NO_ACCESS"""
+        self.sm = 0
+        self.lv = 0
+        
+    def set_open(self):
+        """BLE_GAP_CONN_SEC_MODE_SET_OPEN"""
+        self.sm = 1
+        self.lv = 1
+        
+    def set_enc_no_mitm(self):
+        """BLE_GAP_CONN_SEC_MODE_SET_ENC_NO_MITM"""
+        self.sm = 1
+        self.lv = 2
+        
+    def set_enc_with_mitm(self):
+        """BLE_GAP_CONN_SEC_MODE_SET_ENC_WITH_MITM"""
+        self.sm = 1
+        self.lv = 3
+        
+    def set_lesc_enc_with_mitm(self): 
+        """BLE_GAP_CONN_SEC_MODE_SET_LESC_ENC_WITH_MITM"""
+        self.sm = 1
+        self.lv = 4
+    
+    def set_signed_no_mitm(self):
+        """BLE_GAP_CONN_SEC_MODE_SET_SIGNED_NO_MITM"""
+        self.sm = 2
+        self.lv = 1
+    def set_signed_with_mitm(self):
+        """BLE_GAP_CONN_SEC_MODE_SET_SIGNED_WITH_MITM"""
+        self.sm = 2
+        self.lv = 2
+
+    @classmethod
+    def from_c(cls, sec_mode):
+        return cls(
+            sm=sec_mode.sm,
+            lv=sec_mode.lv,
+        )
+        
+    def to_c(self):
+        sec_mode = driver.ble_gap_conn_sec_mode_t()
+        sec_mode.sm = self.sm
+        sec_mode.lv = self.lv
+        
+        return sec_mode
+
+    def __str__(self):
+        return "sm({0.ssm}) lv({0.lv}))".format(
```

Comment:
Should be sm, not ssm?

Reply:
Great catch! Yes, should be sm. How do we fix this? Should I push a fix?